client/src/components/Chat/Chat.js

```diff
@@ -32,6 +32,7 @@ const Chat = props => {
   const [messageInputError, setMessageInputError] = useState('');
   const [friendLanguage, setFriendLanguage] = useState('');
   const [languageError, setLanguageError] = useState('');
+  const [translation, setTranslation] = useState(true);
```

Comment:
this is a nitpick (minor) but maybe renaming it to `showMessagesInOriginalLanguage` would be more descriptive.

Reply:
Fixed it